s3tests/functional/test_s3.py

```diff
@@ -2359,9 +2359,13 @@ def test_get_object_ifmodifiedsince_failed():
 
     # Sleep since Amazon returns 200 if the date is in the future:
     # https://forums.aws.amazon.com/message.jspa?messageID=325930
-    now = time.time()
-    time.sleep(20)
-    after = formatdate(now + 10)
+
+    for k in bucket.get_all_keys():
+        key = k
```

Comment:
@andrewgaul tried that but it returns a different time format that is harder to work with